dev_utils/docker-compose.yml

```
@@ -115,22 +128,37 @@ services:
       - "8000:8000"
       - "8001:8001"
 
+  database:
+    container_name: db
+    image: neicnordic/sda-db:v2.0.0
+    environment:
+      - DB_LEGA_IN_PASSWORD=inpass
+      - DB_LEGA_OUT_PASSWORD=outpass
+      - PGVOLUME=/var/lib/postgresql
+      - NOTLS=true
+    volumes:
+      - psqldata:/var/lib/postgresql
+    ports:
+      - 5432:5432
```

```suggestion
      - 2345:5432
```
Using the default port in all places will cause problems.